draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -9,37 +9,34 @@
 @import "~@kaizen/component-library/components/Button/styles";
 @import "~@kaizen/deprecated-component-library-helpers/styles/type";
 
-/* The bordered part of the triangle */
-$default-size: 8px;
-$large: 450px;
+// Sync with PopoverModern.tsx
+$arrow-width: 16px;
+$arrow-height: 8px;
+
+$large-width: 450px;
 
 // Suggestion: with this implementation, the anchor point of the popover is at the base of the triangle. But,
 // we would ideally want it at the tip of the triangle. Otherwise, when using the popover, the consumer needs to manually
 // add something like a `margin-top: 8px` to get the popover properly positioned.
 // I didn't update this, as I didn't want to introduce a breaking change.
-@mixin arrow($background-color, $border-color, $size: $default-size) {
-  position: absolute;
-  top: 100%;
-  left: 50%;
-
```

Comment:
Thankfully, I was able to pretty much use the same classes as before. What I did was separate anything that related to popover placement into separate classes. So for the modern popover, I simply didn't use those classes.